src/main/java/com/tstepnik/planner/domain/user/User.java

```diff
@@ -14,15 +15,20 @@
     private Long id;
 
     @NotEmpty
+    @Size(min=3)
     private String login;
 
     private String firstName;
 
     private String lastName;
 
+    @NotEmpty
+    @Size(min=3)
+    @Email
```

Comment:
`@Email` probably includes `@notempty` and `@size` itself. You check it out and see how `@email` acts on its own.

Reply:
Yes, it's include all of them and even more i think.